<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background: rgb(0, 23, 71);
font-family: Microsoft Sans Serif;
}

QPushButton{
background-color: rgb(1, 179, 191);
font-size: 40px;
border-radius: 20px;
color: rgb(0,0,0);
}

QPushButton:hover{
background-color: rgb(77, 242, 243);
}


</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>110</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>LogoSimuladorTesis.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="MainTitleLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
qproperty-alignment: AlignCenter;
font-family: Microsoft Sans Serif;
font-size: 50px;
border-top: 3px solid white;
border-bottom: 3px solid white;
color: white;
}</string>
         </property>
         <property name="text">
          <string>SIMULADOR DE COMUNICACIONES DIGITALES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>135</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="SClasicasButton">
         <property name="minimumSize">
          <size>
           <width>490</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>490</width>
           <height>110</height>
          </size>
         </property>
         <property name="text">
          <string>SEÑALES CLÁSICAS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>135</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="SNoConvencionalesButton">
         <property name="minimumSize">
          <size>
           <width>490</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>490</width>
           <height>110</height>
          </size>
         </property>
         <property name="text">
          <string>SEÑALES NO 
 CONVENCIONALES</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>950</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4"/>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="DocumentationButton">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(255, 177, 64);
font-size: 20px;
border-radius: 20px;
text-align: right;
}

QPushButton:hover{
background-color: rgb(255, 202, 128);
}
</string>
         </property>
         <property name="text">
          <string>Documentación   </string>
         </property>
        </widget>
       </item>
      </layout>
      <widget class="QLabel" name="ManualPic">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>41</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color: rgb(255, 177, 64, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Manual Icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>DocumentationButton</zorder>
      <zorder>ManualPic</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
